sdcm/nemesis.py

```diff
@@ -1548,6 +1548,66 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_cluster_down_scaling(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.terminate
+        2.Full repair
+        3.Nodetool removenode
+        4.Sleep 10 minutes
+        5.Add new node
+        """
+
+        # pick random node
+        self._set_current_disruption('ScaleDown')
+        node = self.target_node
+        up_nodes = self.cluster.get_nodes_up_and_normal(verification_node=node)
+
+        # pick a random node to remove (should be different than node)
+        node_to_remove = random.choice(up_nodes)
+        while node_to_remove is node:
+            node_to_remove = random.choice(up_nodes)
+
+        # get node's host_id
+        status = self.cluster.get_nodetool_status(verification_node=node_to_remove)
+        for dc_status in status.values():
+            ip_status = dc_status.get(node_to_remove.ip_address)
+            if ip_status:
+                host_id = ip_status["host_id"]
+                break
+
+        # node stop and make sure its "DN"
+        node_to_remove.stop_scylla_server(verify_up=True, verify_down=True)
+        # full cluster repair
+        # Use the nodetool repair -pr on each node in the cluster, sequentially.
+        # for node in up_nodes:
+        #     if node is not node_to_remove:
+        #         node.rep("repair -pr")
+        self.repair_nodetool_repair(node=node)
```

Comment:
the idea is todo a full cluster repair, not only one node

Reply:
done